### PR TITLE
Removes all instances isDev from the scala templates

### PR DESCRIPTION
**Pull request by DiegoVazquezNanini**
@TBonnin

### CODE REVIEW

File: common/app/views/support/JavaScriptPage.scala [hunk 1]
```diff
@@ -5,10 +5,9 @@ import common.Maps.RichMap
 import conf.Configuration
 import conf.Configuration.environment
 import model._
-import play.api.Play
-import play.api.Play.current
 import play.api.libs.json.{JsBoolean, JsString, JsValue, Json}
 import play.api.mvc.RequestHeader
+import play.api.Mode.Dev
```
Comment (TBonnin) [inline common/app/views/support/JavaScriptPage.scala]: Is this needed?

File: common/test/views/support/cleaner/AtomCleanerTest.scala [hunk 2]
```diff
@@ -8,8 +8,9 @@ import org.scalatest.{FlatSpec, Matchers}
 import test.TestRequest
 import views.support.AtomsCleaner
 import conf.switches.Switches
+import play.api.Environment
 
-class AtomCleanerTest extends FlatSpec with Matchers with FakeRequests {
+class AtomCleanerTest (implicit env: Environment) extends FlatSpec with Matchers with FakeRequests {
```
Comment (TBonnin) [inline common/test/views/support/cleaner/AtomCleanerTest.scala]: Not using `WithTestEnvironment`?

**Review by TBonnin — CHANGES_REQUESTED** [review]
Thank you very much for doing this @DiegoVazquezNanini 
I made 2 small comments but this PR looks wonderful

**Comment by DiegoVazquezNanini** [comment]
Thanks for those comments. @TBonnin I corrected both cases.

**Review by TBonnin — APPROVED** [review]

**Pull request merged**